files/epub.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "epub.js",
+  "repo": "futurepress/epub.js",
+  "files": {
+    "include": ["build/epub.min.js", "build/hooks.min.js", "build/reader.min.js", "build/libs/localforage.min.js", "build/libs/zip.min.js"]
```

Comment:
local forage is already hosted here http://www.jsdelivr.com/projects/localforage, is your version different?